berkeley-mobile/Calendar/AcademicCalendarViewController.swift

```diff
@@ -80,6 +80,13 @@ extension AcademicCalendarViewController: UITableViewDelegate, UITableViewDataSo
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         tableView.deselectRow(at: indexPath, animated: true)
     }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+//        return UITableView.automaticDimension
+        let entry = calendarEntries[indexPath.row]
+        let charCount = entry.name.count
+        return min(max(50.0 * (CGFloat(charCount) / 30.0), 86), 150)
```

Comment:
I get what we're going for here, I think it would be easier/make more sense to do this in EventTableViewCell by constraining the label.
Try setting a maximum height constraint on the eventName label in EventTableViewCell (`eventName.heightAnchor.constraint(lessThanOrEqualToConstant: 100).isActive = true`) or similar.
In general try to use constraints instead of doing math with constants.